src/components/common/Modal/Modal.tsx

```diff
@@ -69,26 +68,7 @@ const Modal = ({
       </Grid>
       <DialogContent dividers={dividers}>{content}</DialogContent>
       <DialogActions>
-        {customActionSection ? (
-          customActionSection
-        ) : (
-          <>
-            <Button
-              disabled={disableActionButtons}
-              onClick={() => handleClose()}
-            >
-              {cancelText ? cancelText : t('Cancel')}
-            </Button>
-            <Button
-              disabled={disableActionButtons}
-              variant="contained"
-              color="primary"
-              onClick={() => handleConfirm()}
-            >
-              {confirmText ? confirmText : t('Save')}
-            </Button>
-          </>
-        )}
+        {customActionSection ? customActionSection : null}
```

Comment:
So by default there are no buttons? Is the only way out to click the X?

You could just make an "OK" button that is there by default that calls the same handler as the close button.